mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -113,3 +113,9 @@ Description of the change and how users should adjust their configuration (if ne
 ### `deletion_policy` now defaults to `DELETE`
 
 Previously, `google_firebase_web_app` deletions default to `ABANDON`, which means to only stop tracking the WebApp in Terraform. The actual app is not deleted from the Firebase project. If you are relying on this behavior, set `deletion_policy` to `ABANDON` explicitly in the new version.
+
+## Resource: `google_identity_platform_project_default_config`
```

Comment:
@c2thorn This PR is to add fields in `google_identity_platform_project_default_config` to `google_identity_platform_config` and deprecate the `google_identity_platform_project_default_config` resource. Would you mind verifying if we need to add the deprecation message to 5.0 upgrade guide and if there's anything else needed to deprecate a resource?

Reply:
@c2thorn So you're suggesting to move the deprecation message for `google_identity_platform_project_default_config` to another PR?

Reply:
Thanks @c2thorn